### ISSUE TITLE
Updating to Ubuntu 22.04 results in InstanceInfo missing error and how to fix it.

### ISSUE THREAD

**Issue by Uzlopak** [nodkz/mongodb-memory-server#632]
Well, I upgraded to Ubuntu 22.04. So I run mongodb-memory-server and I get only missing InstanceInfo Error. Then I run the downloaded mongo binary and got the actual useful errror that libcrypto.so.1.1 was missing. So I run 

```
wget http://nz2.archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1l-1ubuntu1.2_amd64.deb

sudo dpkg -i libssl1.1_1.1.1l-1ubuntu1.2_amd64.deb
```

ref: https://askubuntu.com/a/1403463

and now mongodb-memory-server works again.

It would have been useful, if mongodb-memory-server gave the actual error. I fixed it for me, but as ubuntu 22.04 is released yesterday, I assume that these bug reports could come more frequently.

Thank you.

**Comment by hasezoey** [comment]
Thanks for providing feedback about a updated distro

> So I run mongodb-memory-server and I get only missing InstanceInfo Error.

this should be fixed in the next patch update (in 8.5.1)

> Then I run the downloaded mongo binary and got the actual useful errror that libcrypto.so.1.1 was missing

currently there are no mongodb builds for any ubuntu version above 2004 (eg 2104 or 2204), meaning we have to fallback to 2004, which may mean that older libraries have to be installed

---

also, please dont remove the issue template

**Comment by Uzlopak** [comment]
That was a fast implementation :)

**Comment by jlimas** [comment]
Just adding that I found this error today after upgrading but I was getting the following error
`Instance Exited before being ready and without throwing an error!`
@Uzlopak instructions did fix the problem, thanks!

**Comment by hasezoey** [comment]
@jlimas what version of MMS were you using?

**Comment by jlimas** [comment]
Before I noticed the issue I was using 
`"mongodb-memory-server": "^8.0.4",`
After that first thing I tried was using the latest 8.5.2 but kept getting the same error even after a npm ci

**Comment by hasezoey** [comment]
> Before I noticed the issue I was using `"mongodb-memory-server": "^8.0.4",` After that first thing I tried was using the latest 8.5.2 but kept getting the same error even after a npm ci

still the same error with 8.5.2? since 8.5.1 there should have been a error message (no error showing should have been fixed in 8.5.1)

**Comment by tjsearsz** [comment]
The version of the lib got updated in the meantime, latest url to download is: `http://nz2.archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1l-1ubuntu1.4_amd64.deb`
